playground/infrastructure/test_cd_helper.py

```diff
@@ -13,16 +13,83 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import mock
+import os
+import shutil
+
+import pytest
 
 from cd_helper import CDHelper
+from api.v1.api_pb2 import SDK_JAVA, STATUS_UNSPECIFIED
+from config import Config
+from helper import Example, Tag
+
+
+@pytest.fixture
+def delete_temp_folder():
+    """Create temp folder for tests with storing files"""
+    yield delete_temp_folder
+    if os.path.exists(Config.TEMP_FOLDER):
+        shutil.rmtree(Config.TEMP_FOLDER)
+
+
+def upload_blob():
+    """
+    Fake method for mocking
+    Returns: None
+
```

Comment:
blank line

Reply:
it is a way the pycharm generate comments

Reply:
I'll fix it